systemd/hitch.conf

```diff
@@ -43,26 +43,29 @@ ocsp-dir = "/var/lib/hitch-ocsp"
 # Verification of OCSP responses
 # ocsp-verify-staple = off
 
-# SSL protocol.
+# List of allowed cipher TLS suites
 #
-# tls = on
-# ssl = off
+# type: string
+ciphersuites = "TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256"
 
-# List of allowed SSL ciphers.
+# List of allowed TLS ciphers.
 #
-# Run openssl ciphers for list of available ciphers.
 # type: string
-ciphers = "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH"
+ciphers = "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"
 
 # Enforce server cipher list order
 #
 # type: boolean
 prefer-server-ciphers = off
 
-# Use specified SSL engine
+# The SSL/TLS protocols to be used
 #
 # type: string
-ssl-engine = ""
+tls-protos = TLSv1.2 TLSv1.3
+
+# Enable to let clients negotiate HTTP/2 with ALPN. (default off)
+# type: string
+#alpn-protos = "h2, http/1.1"
```

Comment:
any benefit to not eabling it?

Reply:
we touched the subject during intwash, and i'm thinking more and more that we should just embrace the H/2 train. Enable it in the VC/VCP package and now here. This is something that the crowd wants and we'll need to accommodate, so we might as well prepare the packages for it

Reply:
relevant to this discussion: https://github.com/varnishcache/pkg-varnish-cache/pull/143

Reply:
Enabling this by default would break setups where they didn't enable h/2 in Varnish.

I'm OK with this as long as we also get https://github.com/varnishcache/pkg-varnish-cache/pull/143 merged.

Reply:
merged!